### PR TITLE
Register Subdomain: rlewsley

### PR DESCRIPTION
**Pull request by lewsley-r**
Added the domain: rlewsley

### CODE REVIEW

File: domains/rlewsley.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+      "owner": {
+        "username": "lewsley-r",
+        "email": "lewsley-r@ulster.ac.uk",
+        "name": "Ronan Lewsley",
+        "role": "junior full stack dev"
+        "github-link": "https://github.com/lewsley-r"
+      },
+
+      "record": {
+        "CNAME": ["https://lewsley-r.github.io"]
```
Comment (WilliamDavidHarrison) [inline domains/rlewsley.json:12]: CNAME cannot be an array or have `https://`.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/rlewsley.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+
+    {
+      "owner": {
+        "username": "lewsley-r",
+        "email": "lewsley-r@ulster.ac.uk",
+        "name": "Ronan Lewsley",
```
Comment (WilliamDavidHarrison) [inline domains/rlewsley.json:6]: You don't need this field.

File: domains/rlewsley.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+        "role": "junior full stack dev"
```
Comment (WilliamDavidHarrison) [inline domains/rlewsley.json:7]: You don't need this field.

File: domains/rlewsley.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+        "github-link": "https://github.com/lewsley-r"
```
Comment (WilliamDavidHarrison) [inline domains/rlewsley.json:8]: You don't need this field.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/rlewsley.json (hunk 2 above)
Comment (lewsley-r) [reply domains/rlewsley.json]: removed.

File: domains/rlewsley.json (hunk 3 above)
Comment (lewsley-r) [reply domains/rlewsley.json]: removed.

**Review by wdhdev — DISMISSED** [review]

**Review by MaskDuck — APPROVED** [review]

**Pull request merged**